jwst/assign_wcs/nirspec.py

```diff
@@ -72,7 +72,7 @@ def imaging(input_model, reference_files):
     dircos2unitless = DirCos2Unitless(name='directional_cosines2unitless')
 
     col_model = CollimatorModel(reference_files['collimator'])
-    col = col.model
+    col = col_model.model
```

Comment:
I don't think it will fix the remaining 3 tests in `assign_wcs` that are failing, but perhaps those tests need to be re-written, as they currently test against Build 6 versions of the reffiles.  From what I can see, `test_correct_tilt`, `test_nirspec_fs_esa` and `test_nirspec_ifu_against_esa` seem to compare to old, non-CV3 reffiles?

Should these tests be disabled or is it possible to rewrite them?  Or perhaps disabled temporarily?